addons/hr_expense/models/hr_expense.py

```diff
@@ -959,6 +1630,42 @@ def _prepare_payments_vals(self):
         }
         return move_vals, payment_vals
 
+    def _prepare_move_vals(self):
+        return {
+            # force the name to the default value, to avoid an eventual 'default_name' in the context
+            # to set it to '' which cause no number to be given to the account.move when posted.
```

Comment:
that's unclear to me. Do you mean the following?
```suggestion
            # that would set it to ''... which cause no number to be given to the account.move when posted.
```